weaviate/users/user_db_lister.go

```diff
@@ -0,0 +1,42 @@
+package users
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/weaviate/weaviate-go-client/v5/weaviate/connection"
+	"github.com/weaviate/weaviate-go-client/v5/weaviate/except"
+	"github.com/weaviate/weaviate/client/users"
+)
+
+type UserInfoList []UserInfo
+
+type UserDbLister struct {
```

Comment:
```suggestion
type UserDBLister struct {
```

suggestion: [Initialisms](https://google.github.io/styleguide/go/decisions.html#initialisms) should have consistent case

Not to spam the PR with identical comments, also applies to UserDbActivator, UserDbDeactivator, UserDbGetter, etc.